EIPS/eip-2135.md

```diff
@@ -0,0 +1,74 @@
+
+---
+eip: 2135
+title: Consumable Interface
+author: Zainan Victor Zhou (@xinbenlv)
+discussions-to: https://github.com/ethereum/EIPs/issues/2135
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-06-23
+requires (*optional): N/A
```

Comment:
If this is N/A then you can leave it out

Reply:
DONE